DEPLOY.md

```diff
@@ -0,0 +1,44 @@
+# Deploying Atlas
+
+This is a guide to getting the Atlas smart contracts deployed on an EVM chain.
+
+## Atlas Deployment Process
+
+1. Check that the contracts compile and all tests pass.
```

Comment:
Step 1 should be checking out a tag to deploy I think, have we tagged the spearbit audit hash and documented the hash that was deployed to polygon somewhere?

Reply:
Added some more on the commit hash and latest deployment addresses here https://github.com/FastLane-Labs/atlas/pull/378/commits/be188c4d6bc50f4c259440802c38191c4c9d7692